### DISCUSSION TITLE
[0.1300.0.0] Radiation Feedback and changes

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#5129] [Unstable feedback]
So, let's talk about the wall of radiation:

**The Good**
- Radiation does its job, which is pushing players forward so they don't hang around too long in the low-difficulty areas. You _shouldn't_ be able to accumulate resources and max out sub upgrades too easily before moving on
- It isn't instantly lethal, so you can still go through irradiated areas without dying if need be

**The Meh:**
- All it does is burn you over time. Between this and ballast flora, hazards are synonymous with bandage spam. Radiation needs some more spice to it. Disrupt sonar/visibility/radio? Add enemies?

**The Bad:**
- The wall is only a "wall" that can be outpaced in one small sprint and entirely forgotten for the rest of the game. 

Things to learn from FTL, and the "rebel fleet" approach:
![Capture](https://user-images.githubusercontent.com/73229309/108649879-8510a180-74be-11eb-84d5-5d18acfd7716.JPG)

If you're not familiar with the game, FTL has a somewhat similar mechanic in theory; a wall of advancing "danger" forcing the players to push forward (a fleet of enemies that are *much* stronger than the player which I will refer to as "wall"). Sorry you haven't played the game, because i'll reference a few subtleties down below

There are three things that make the rebel fleet approach more interesting than the radiation wall:

1. **Manageable risk**: the ship can be equipped with ways to "endure" a fight long enough to escape without sustaining too much damage. If you have a cloak, a good missile, defense drones, hacking or simply good engines... you have many options. In Baro, radiation is just several crates of bandages and some antirad. Not too exciting.  We need more options to deal with "the wall", which implies a more interesting wall (the Meh). What if rad didn't progress in the deeper parts of Europa? (see #4971 ) 
![Capture2](https://user-images.githubusercontent.com/73229309/108651702-a4a9c900-74c2-11eb-9e87-52274b3dbfb9.JPG)

2. **The sector segmentation**: the great advantage FTL's wall has over Baro is that the game is divided in "sectors", with gradual difficulty increases with each sector. This means two things: It's worth staying in a sector for as long as possible, and the "wall" resets every sector. It'll be a somewhat lengthy explanation but hang on:

The longer you stay in sector, the more resources you'll accumulate and the stronger you will be relative to that sector. It's worth staying as long as possible in that sector, which means doing as many nodes as you can; and to optimize that, you can "Dive". Diving means that, instead of exiting the sector before the wall catches up, you go beyond the exit to clear 1-2 extra nodes for rewards, and go to the exit at the last possible moment to move to the next sector. You are risking one, hard fight in exchange for extra rewards, so there's meaningful strong risk-reward component. In Baro, Radiation doesn't have much of the risk nor the reward right now. I don't really see a reason anyone should go back through radiation.

Finally, the sector "reset" means that if you either sprinted to the exit, or took your time to dive, it wouldn't matter by the next sector, as the position of your ship and the wall are reset. This means there is no long-term pacing problem of either being completely caught by the wall (running out of fuel), or straight up outpacing the wall (doing nebulas). New sector = fresh wall. In Baro however it will be hard to find a balance between too fast and too slow since the progress is constant across the entire campaign. 

A good improvement would be having both the wall approaching from behind, but also "patches" of radiation appearing ahead of the wall, with increased density the closer to the wall you are: 
![Capture3](https://user-images.githubusercontent.com/73229309/108652957-a2953980-74c5-11eb-8103-00e191718370.JPG)

Even if you outpace the wall, radiation could spontaneously appear, and it would be actually dangerous to stay close to the wall in the easier areas (in both difficulty and high outpost reputation). Also, with radiation appearing beyond the wall, it may provide you with the actual choice of dealing with it should you choose a mission or path through irradiated areas.

**Comment by mLuby** [comment]
You make a good point about sector resets. Barotrauma has different biomes, somewhat visible on the map and outside the sub, that could be places for soft resets (i.e. until the sub enters biomes, radiation doesn't leak into them, or does so slowly). [![biomes](https://barotraumagame.com/baro-wiki/images/a/a7/Europa_cross_section.png)](https://barotraumagame.com/wiki/Biome)
